website/docs/getting-started/getting-started.md

```diff
@@ -347,7 +358,71 @@ The generation request accepts these parameters:
 - `use_source_context`: Optional boolean (default: true) that determines whether to use source analysis for generation
 
 2. Check generation status:
+
+The response will include generation id and version id:
+```json
+{
+    "tenant_id": "your-tenant-id",
+    "result": null,
+    "worker_id": "your-worker-id",
+    "output_type": "your-output-type",
+    "status": "PROCESSING",                              // NOT_STARTED, QUEUED, PROCESSING, COMPLETED, FAILED
+    "prompt": "your-worker-prompt",
+    "source_ids": [],
+    "created_at": "2025-02-14T15:49:44.120298+00:00",
+    "context": "your-context",
+    "generation_id": "your-generation-id",               // generation id
+    "current_version_id": "latest-version-id",           // version id
+    "updated_at": "2025-02-14T15:49:44.120298+00:00",
+    "use_source_context": true,
+    "brand_id": "your-brand-id"
+}
+```
+
+Fetch the generation version using the generation id and version id:
+```http
+GET https://social-toolkit.ti.trilogy.com/tenant/{tenant_id}/brand/{brand_id}/worker/{worker_id}/generation/{generation_id}/version/{version_id}
+Authorization: Bearer <your-tenant-api-key>
+```
+
+Here is the response for the generation version:
+```json
+{
+    "result": null,
+    "feedback": null,
+    "previous_version_id": null,
+    "version_id": "your-version-id",
+    "status": "PROCESSING",                               // NOT_STARTED, QUEUED, PROCESSING, COMPLETED, FAILED
+    "generation_id": "your-generation-id",
+    "created_at": "2025-02-14T15:49:44.120298+00:00",
+    "output_type": "your-output-type"
+}
+```
+
+You can fetch the generation as well, using the generation id but not that this will return the latest version of the generation.
```

Comment:
```suggestion
You can fetch the generation as well, using the generation id but note that this will return the latest version of the generation.
```